README.md

```diff
@@ -1,6 +1,13 @@
-# React Flagpack
+# Flagpack
 
-> A project made by Yummygum
+Flagpack contains 260+ flag icons to easily use within your code project. Flagpack is an open source project and available for JavaScript frameworks/libraries Angular, Vue and React.
+
+![Flagpack. 260+ easily implementable flag icons to use in your design or code project. Open Source. Available for Sketch, Figma, Angular, Vue, and React. [www.flagpack.xyz](http://www.flagpack.xyz). Made with love by Yummygum. Graphic showing a list with the flags of Argentina, Croatia, Estonia, Kenia, Netherlands, and Scotland.](https://flagpack.xyz/meta-image.png)
+
+[View documentation on flagpack.xyz](https://flagpack.xyz/docs/)
+
+## react-flagpack
+Flagpack for React is created using React v16.8. The Flagpack component has not been tested for compatibility with older version of React.
 
 ## Installation
```

Comment:
It's a bit nitpicky, but should we set an example to run the project with `yarn` aswell? Yarn is apparently [pretty close in usage](https://www.npmtrends.com/npm-vs-yarn) compared to npm.